frontend/src/App.res

```diff
@@ -18,19 +18,44 @@ query ($startDate: timestamp!, $endDate: timestamp!) {
   }
 `)
 
-let comparePulls = ((pn1, _b1), (pn2, _b2)) => {
-  -compare(pn1, pn2)
+module PullCompare = Belt.Id.MakeComparable({
+  type t = (int, option<string>)
+  let cmp = compare
```

Comment:
Note that this seems to drop the `-` which was there to unsure that the PRs are listed from the most recent to the oldest.

Reply:
Oops, that was not intentional. Good catch 👍